catkin_ws/src/image_rotation/CMakeLists.txt

```diff
@@ -123,6 +126,9 @@ include_directories(
   ${catkin_INCLUDE_DIRS}
 )
 
+include_directories(${OpenCV_INCLUDE_DIRS})
+include_directories(include/image_rotation/)
```

Comment:
Change it to "include/" and make corresponding changes in inclusions to be consistent with the subsciber

Reply:
done